cli/auth.go

```diff
@@ -112,20 +123,27 @@ func (c *viamClient) loginAction(cCtx *cli.Context) error {
 			already = ""
 			viamLogo(cCtx.App.Writer)
 		}
-
 		fmt.Fprintf(cCtx.App.Writer, "%slogged in as %q, expires %s\n", already, t.User.Email,
 			t.ExpiresAt.Format("Mon Jan 2 15:04:05 MST 2006"))
 	}
 
-	if c.conf.Auth != nil && !c.conf.Auth.isExpired() {
-		loggedInMessage(c.conf.Auth, true)
+	if _, isAPIKey := c.conf.Auth.(*token); isAPIKey {
+		warningf(c.c.App.Writer, "was logged in with an api-key. logging out")
```

Comment:
[minor / ignorable] worth adding the key ID?

Reply:
I thought about this but didn't add because it made the login messages too noisy and it caused me to start skimming. 

Happy to revert if others think it would be better to print the key here.